.go-version

```diff
@@ -1 +1 @@
-1.14.1
+1.14.10
```

Comment:
Is there any particular reason behind this bump?

Also can we limit this to versions which are available from goenv? 1.14.10 is not available there yet.
https://github.com/syndbg/goenv/pull/149